app/src/main/java/net/squanchy/service/firestore/FirestoreDbService.kt

```diff
@@ -17,10 +21,10 @@ class FirestoreDbService(private val db: FirebaseFirestore) {
 
     fun scheduleView(): Observable<List<FirestoreSchedulePage>> {
         return Observable.create { subscriber ->
-            val registration = db.collection("views")
-                .document("schedule")
-                .collection("schedule_pages")
-                .orderBy("day.date")
+            val registration = db.collection(VIEWS)
```

Comment:
maybe we could extract `db.collection(VIEWS).document(SCHEDULE)` as a method for brevity:

```kotlin
fun FirestoreDbService.view(viewName: String) = db.collection(VIEWS).document(viewName)
```